#VRML_SIM R2021a utf8
WorldInfo {
  info [
    ""
  ]
  coordinateSystem "NUE"
}
Viewpoint {
  orientation -0.9999702503439193 -0.006928943414877459 0.00338942034462967 1.006889061584209
  position -2.6002887145437774 5.191300521906995 0.4668722059217059
  follow "E-PUCK1"
}
TexturedBackground {
}
TexturedBackgroundLight {
}
RectangleArena {
  name "rectangle arena(1)"
  floorSize 10 10
  floorTileSize 2 2
  wallHeight 0
}
DEF TURNSSUPERVISOR Robot {
  name "TURNS_SUPERVISOR"
  supervisor TRUE
}
DEF MAINSUPERVISOR Robot {
  children [
    Emitter {
      channel 5
    }
    Receiver {
      name "receiver1"
      channel 1
    }
    Receiver {
      name "receiver2"
      channel 2
    }
    Receiver {
      name "receiver3"
      channel 3
    }
    Receiver {
      name "receiver4"
      channel 4
    }
  ]
  name "MAIN-SUPERVISOR"
  controller "my_supervisor_controller"
  supervisor TRUE
}
DEF Laptop Robot {
  translation -0.4835069992426404 -0.0011383680366373813 -0.6479854092861486
  rotation 1.5163876721829265e-06 0.9999991506711651 -0.0013033244604998418 3.141589998872823
  children [
    HingeJoint {
      jointParameters HingeJointParameters {
        position 4.064684964007496e-06
        anchor 0 0.0142 0.0546
        minStop -2.105
        maxStop 0.01
        staticFriction 0.8
      }
      device [
        PositionSensor {
        }
      ]
      endPoint Solid {
        translation 0.014499999999999888 0.12999981546234513 0.1000004706901429
        rotation 1 0 0 4.0647133925121234e-06
        children [
          Shape {
            appearance PBRAppearance {
              baseColorMap ImageTexture {
                url [
                  "textures/laptop_screen_base_color.jpg"
                ]
              }
              roughnessMap ImageTexture {
                url [
                  "textures/laptop_screen_roughness.jpg"
                ]
              }
              metalness 0
              normalMap ImageTexture {
                url [
                  "textures/laptop_screen_normal.jpg"
                ]
              }
            }
            geometry IndexedFaceSet {
              coord Coordinate {
                point [
                  0.15393 -0.121846 -0.030726
                  0.15393 0.068025 0.081871
                  -0.192175 0.068025 0.081871
                  -0.192175 -0.121846 -0.030726
                  0.160877 -0.121901 -0.042869
                  0.160877 0.079694 0.07668
                  -0.199123 0.079694 0.07668
                  -0.199123 -0.121901 -0.042869
                  0.098279 -0.122127 -0.049684
                  0.098279 -0.122126 -0.032252
                  0.083065 -0.122126 -0.032252
                  0.083065 -0.122127 -0.049684
                  0.098279 -0.107256 -0.049684
                  0.098279 -0.107256 -0.032252
                  0.083065 -0.107256 -0.032252
                  0.083065 -0.107256 -0.049684
                  -0.115506 -0.122127 -0.049684
                  -0.115506 -0.122126 -0.032252
                  -0.13072 -0.122126 -0.032252
                  -0.13072 -0.122127 -0.049684
                  -0.115506 -0.107256 -0.049684
                  -0.115506 -0.107256 -0.032252
                  -0.13072 -0.107256 -0.032252
                  -0.13072 -0.107256 -0.049684
                ]
              }
              texCoord TextureCoordinate {
                point [
                  0.936 0.5783, 0.919 0.5783, 0.9356 0.5696, 0.9237 0.5632, 0.9313 0.5632, 0.9238 0.5658, 0.9126 0.5577, 0.9174 0.5577, 0.9128 0.5601, 0.9303 0.5437, 0.9197 0.5437, 0.9301 0.5401, 0.919 0.5965, 0.919 0.5881, 0.9321 0.5965, 0.9795 0.3669, 0.9795 0.9912, 0.0205 0.9912, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.919 0.5783, 0.9195 0.5696, 0.9356 0.5696, 0.9313 0.5632, 0.9312 0.5658, 0.9238 0.5658, 0.9174 0.5577, 0.9172 0.5601, 0.9128 0.5601, 0.9197 0.5437, 0.9199 0.5401, 0.9301 0.5401, 0.919 0.5881, 0.9321 0.5881, 0.9321 0.5965, 0.0205 0.3669, 0.9795 0.3669, 0.0205 0.9912, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436, 0.0075 0.435, 0.0075 0.435, 0.008 0.436
                ]
              }
              coordIndex [
                4, 5, 0, -1, 5, 6, 1, -1, 6, 7, 2, -1, 7, 4, 3, -1, 0, 1, 3, -1, 7, 6, 5, -1, 12, 13, 9, -1, 13, 14, 9, -1, 14, 15, 11, -1, 15, 12, 11, -1, 8, 9, 11, -1, 15, 14, 12, -1, 20, 21, 16, -1, 21, 22, 17, -1, 22, 23, 19, -1, 23, 20, 16, -1, 16, 17, 19, -1, 23, 22, 21, -1, 5, 1, 0, -1, 6, 2, 1, -1, 7, 3, 2, -1, 4, 0, 3, -1, 1, 2, 3, -1, 4, 7, 5, -1, 8, 12, 9, -1, 14, 10, 9, -1, 10, 14, 11, -1, 12, 8, 11, -1, 9, 10, 11, -1, 14, 13, 12, -1, 21, 17, 16, -1, 22, 18, 17, -1, 18, 22, 19, -1, 19, 23, 16, -1, 17, 18, 19, -1, 20, 23, 21, -1
              ]
              texCoordIndex [
                0, 1, 2, -1, 3, 4, 5, -1, 6, 7, 8, -1, 9, 10, 11, -1, 12, 13, 14, -1, 15, 16, 17, -1, 18, 19, 20, -1, 21, 22, 23, -1, 24, 25, 26, -1, 27, 28, 29, -1, 30, 31, 32, -1, 33, 34, 35, -1, 36, 37, 38, -1, 39, 40, 41, -1, 42, 43, 44, -1, 45, 46, 47, -1, 48, 49, 50, -1, 51, 52, 53, -1, 54, 55, 56, -1, 57, 58, 59, -1, 60, 61, 62, -1, 63, 64, 65, -1, 66, 67, 68, -1, 69, 70, 71, -1, 72, 73, 74, -1, 75, 76, 77, -1, 78, 79, 80, -1, 81, 82, 83, -1, 84, 85, 86, -1, 87, 88, 89, -1, 90, 91, 92, -1, 93, 94, 95, -1, 96, 97, 98, -1, 99, 100, 101, -1, 102, 103, 104, -1, 105, 106, 107, -1
              ]
              creaseAngle 2
            }
          }
          Display {
            translation -0.014 -0.127 -0.101
            children [
              Shape {
                appearance PBRAppearance {
                  baseColorMap ImageTexture {
                  }
                  roughness 0.25
                  metalness 0
                }
                geometry IndexedFaceSet {
                  coord Coordinate {
                    point [
                      0.163694 0.187436 0.165589
                      -0.17284 0.026052 0.070159
                      -0.172755 0.186595 0.165139
                      0.166293 0.027674 0.071251
                    ]
                  }
                  texCoord TextureCoordinate {
                    point [
                      0 1
                      1 0
                      1 1
                      0 0
                    ]
                  }
                  coordIndex [
                    2, 0, 1, -1, 0, 3, 1, -1
                  ]
                }
              }
            ]
            width 620
            height 400
          }
        ]
        boundingObject Transform {
          translation -0.02 -0.023 0.022
          rotation 1 0 0 0.532
          children [
            Box {
              size 0.359 0.232 0.011
            }
          ]
        }
        physics Physics {
          density -1
          mass 0.5
        }
      }
    }
    Shape {
      appearance PBRAppearance {
        baseColorMap ImageTexture {
          url [
            "textures/laptop_keyboard_base_color.jpg"
          ]
        }
        roughness 0.7
        metalness 0
        normalMap ImageTexture {
          url [
            "textures/laptop_keyboard_normal.jpg"
          ]
        }
        occlusionMap ImageTexture {
          url [
            "textures/laptop_keyboard_occlusion.jpg"
          ]
        }
      }
      geometry IndexedFaceSet {
        coord Coordinate {
          point [
            0.154016 3e-06 -0.159485
            0.154016 3e-06 0.042213
            -0.162222 3e-06 0.042213
            -0.162222 3e-06 -0.159485
            0.175897 0.019796 -0.175825
            0.175897 0.019796 0.058553
            -0.184103 0.019796 0.058553
            -0.184103 0.019796 -0.175825
          ]
        }
        texCoord TextureCoordinate {
          point [
            0.2672 0.5894, 0.0609 0.5894, 0.2528 0.3878, 0.0317 0.3946, 0.2808 0.3946, 0.0468 0.5528, 0.2623 0.5999, 0.0384 0.5999, 0.0541 0.3811, 0.2622 0.5458, 0.0659 0.5458, 0.2503 0.4212, 0.0151 0.5659, 0.0151 0.3884, 0.2935 0.5659, 0.9927 0.3583, 0.9927 0.9998, 0.0073 0.3583, 0.0609 0.5894, 0.0753 0.3878, 0.2528 0.3878, 0.2808 0.3946, 0.2657 0.5528, 0.0468 0.5528, 0.2467 0.3811, 0.2623 0.5999, 0.0541 0.3811, 0.0659 0.5458, 0.0778 0.4212, 0.2503 0.4212, 0.0151 0.3884, 0.2935 0.3884, 0.2935 0.5659, 0.9927 0.9998, 0.0073 0.9998, 0.0073 0.3583
          ]
        }
        coordIndex [
          4, 5, 0, -1, 5, 6, 1, -1, 6, 7, 3, -1, 7, 4, 3, -1, 0, 1, 3, -1, 7, 6, 4, -1, 5, 1, 0, -1, 6, 2, 1, -1, 2, 6, 3, -1, 4, 0, 3, -1, 1, 2, 3, -1, 6, 5, 4, -1
        ]
        texCoordIndex [
          0, 1, 2, -1, 3, 4, 5, -1, 6, 7, 8, -1, 9, 10, 11, -1, 12, 13, 14, -1, 15, 16, 17, -1, 18, 19, 20, -1, 21, 22, 23, -1, 24, 25, 26, -1, 27, 28, 29, -1, 30, 31, 32, -1, 33, 34, 35, -1
        ]
        creaseAngle 2
      }
    }
  ]
  name "LAPTOP"
  model "laptop"
  boundingObject Transform {
    translation -0.005 0.01 -0.058
    children [
      Box {
        size 0.36 0.02 0.233
      }
    ]
  }
  physics Physics {
    density 1200
  }
  recognitionColors [
    1 0 0
  ]
  controller "laptop_switch_on"
}
DEF EPUCK1 E-puck {
  translation -3.5 -8.53891e-05 -3.5
  rotation 0 -1 0 5.307156870259904e-06
  name "E-PUCK1"
  controller "void"
  receiver_channel 5
  turretSlot [
    GPS {
      translation 0 0.00025 0
    }
    DistanceSensor {
      translation -7.34639e-08 -0.009 -0.04
      rotation 0 1 0 1.5708
      name "up_sensor"
      lookupTable [
        0 0 0
        10000 10000 0
      ]
    }
    DistanceSensor {
      translation 2.34464e-07 0.001 0.02
      rotation 0 1 0 -1.5707953071795862
      name "down_sensor"
      lookupTable [
        0 0 0
        10000 10000 0
      ]
    }
    DistanceSensor {
      translation 0.033 -0.006 -0.01
      rotation 0 -1 0 -5.307179586466759e-06
      name "right_sensor"
      lookupTable [
        0 0 0
        10000 10000 0
      ]
    }
    DistanceSensor {
      translation -0.033 -0.006 -0.00999973
      rotation 0 1 0 -3.1415853071795863
      name "left_sensor"
      lookupTable [
        0 0 0
        10000 10000 0
      ]
    }
    Camera {
      translation 0 0 -0.035
      name "colour_camera"
      recognitionColors [
        1 0 0
      ]
      fieldOfView 0.7854
      width 250
      height 250
      recognition Recognition {
      }
    }
  ]
  groundSensorsSlot [
    Compass {
      rotation 0 1 0 3.14159
    }
  ]
}
DEF EPUCK2 E-puck {
  translation 4.33209 -6.39203e-05 -0.509865
  rotation 0.000323569999714327 3.729950861472446e-11 0.9999999476512264 6.061970373266844e-06
  name "E-PUCK2"
  controller "void"
  turretSlot [
    GPS {
      translation 0 0.00025 0
    }
    DistanceSensor {
      translation -7.34639e-08 -0.009 -0.04
      rotation 0 1 0 1.5708
      name "up_sensor"
      lookupTable [
        0 0 0
        10000 10000 0
      ]
    }
    DistanceSensor {
      translation 2.34464e-07 0.001 0.02
      rotation 0 1 0 -1.5707953071795862
      name "down_sensor"
      lookupTable [
        0 0 0
        10000 10000 0
      ]
    }
    DistanceSensor {
      translation 0.033 -0.006 -0.01
      rotation 0 -1 0 -5.307179586466759e-06
      name "right_sensor"
      lookupTable [
        0 0 0
        10000 10000 0
      ]
    }
    DistanceSensor {
      translation -0.033 -0.006 -0.00999973
      rotation 0 1 0 -3.1415853071795863
      name "left_sensor"
      lookupTable [
        0 0 0
        10000 10000 0
      ]
    }
    Camera {
      translation 0 0 -0.035
      name "colour_camera"
      recognitionColors [
        1 0 0
      ]
      fieldOfView 0.7854
      width 250
      height 250
      recognition Recognition {
      }
    }
  ]
  groundSensorsSlot [
    Compass {
      rotation 0 1 0 3.14159
    }
  ]
}
DEF mybot Robot {
  translation 1.16525 0.0397998 -4.39262
  rotation -0.562737841661225 0.11854091644811797 -0.8180917874481483 -1.8262004054017278e-07
  children [
    DEF WHEEL HingeJoint {
      jointParameters HingeJointParameters {
        position 2.2102755180892345e-11
        anchor 0.06 0 -0.05
      }
      device [
        RotationalMotor {
        }
      ]
      endPoint DEF Wheels Solid {
        translation 0.06 0 -0.05
        rotation 5.304632593462889e-10 -5.30465207850218e-10 1 1.5708
        children [
          DEF Wheel Shape {
            appearance PBRAppearance {
              baseColor 1 1 0
            }
            geometry Cylinder {
              height 0.02
              radius 0.04
            }
          }
        ]
        name "solid(3)"
        boundingObject USE Wheel
        physics Physics {
        }
      }
    }
    DEF WHEEL HingeJoint {
      jointParameters HingeJointParameters {
        position 2.875185241059213e-07
        anchor 0.06 0 -0.05
      }
      device [
        RotationalMotor {
        }
      ]
      endPoint DEF Wheels Solid {
        translation -0.06 0 -0.05
        rotation 1.437587339966735e-07 -1.4375926205295958e-07 0.9999999999999793 1.5708000000000204
        children [
          DEF Wheel Shape {
            appearance PBRAppearance {
              baseColor 1 1 0
            }
            geometry Cylinder {
              height 0.02
              radius 0.04
            }
          }
        ]
        name "solid(2)"
        boundingObject USE Wheel
        physics Physics {
        }
      }
    }
    DEF WHEEL HingeJoint {
      jointParameters HingeJointParameters {
        position -2.8835377818836605e-07
        anchor 0.06 0 -0.05
      }
      device [
        RotationalMotor {
        }
      ]
      endPoint DEF Wheels Solid {
        translation -0.05999999999999814 2.883537781883581e-08 0.04999999999999445
        rotation -1.4417635950386992e-07 1.4417688909418104e-07 0.9999999999999793 1.5708000000000208
        children [
          DEF Wheel Shape {
            appearance PBRAppearance {
              baseColor 1 1 0
            }
            geometry Cylinder {
              height 0.02
              radius 0.04
            }
          }
        ]
        name "solid(1)"
        boundingObject USE Wheel
        physics Physics {
        }
      }
    }
    DEF WHEEL HingeJoint {
      jointParameters HingeJointParameters {
        position -1.4714455165432128e-07
        anchor 0.06 0 -0.05
      }
      device [
        RotationalMotor {
        }
      ]
      endPoint DEF Wheels Solid {
        translation 0.06 1.4714455165432211e-08 0.04999999999999985
        rotation -7.357200558159767e-08 7.357227582716037e-08 0.9999999999999946 1.570800000000005
        children [
          DEF Wheel Shape {
            appearance PBRAppearance {
              baseColor 1 1 0
            }
            geometry Cylinder {
              height 0.02
              radius 0.04
            }
          }
        ]
        boundingObject USE Wheel
        physics Physics {
        }
      }
    }
    DEF BODY Shape {
      appearance PBRAppearance {
        baseColor 1 0 1
      }
      geometry Box {
        size 0.1 0.03 0.2
      }
    }
  ]
  boundingObject USE BODY
  physics Physics {
  }
}
Wall {
  translation 4 -1.06581e-14 0.5
  rotation 0 1 0 1.5708
  name "wall(3)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 4 -8.88178e-15 1.5
  rotation 0 1 0 1.5708
  name "wall(8)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 0 0 0.5
  rotation 0 1 0 1.5708
  name "wall(18)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 4 -1.77636e-15 -0.5
  rotation 0 1 0 1.5708
  name "wall(24)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -2 1.77636e-15 -3.5
  rotation 0 1 0 1.5708
  name "wall(25)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -2 0 -2.5
  rotation 0 1 0 1.5708
  name "wall(28)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 0 0 -2.5
  rotation 0 1 0 1.5708
  name "wall(29)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 4 0 2.5
  rotation 0 1 0 1.5708
  name "wall(5)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -2 0 0.5
  rotation 0 1 0 1.5708
  name "wall(15)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 4 0 -1.5
  rotation 0 1 0 1.5708
  name "wall(38)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 4 0 -2.5
  rotation 0 1 0 1.5708
  name "wall(42)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 4 -3.55271e-15 -3.5
  rotation 0 1 0 1.5708
  name "wall(43)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 4 0 3.5
  rotation 0 1 0 1.5708
  name "wall(41)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -2 0 -0.5
  rotation 0 1 0 1.5708
  name "wall(31)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -3.5 -7.10543e-15 -1
  name "wall(1)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -2.5 0 4
  name "wall(4)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -3.5 0 4
  name "wall(52)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -1.5 0 4
  name "wall(50)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -0.5 0 3
  name "wall(11)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -2.5 5.32907e-15 -4
  name "wall(51)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -3.5 5.32907e-15 -4
  name "wall(53)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 0 -2.91323e-13 2.5
  rotation 0 1 0 -1.5707953071795862
  name "wall(13)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 2 -2.91323e-13 2.5
  rotation 0 1 0 -1.5707953071795862
  name "wall(58)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -1.5 9.59233e-14 -4
  name "wall(21)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -0.5 0 4
  name "wall(6)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -3.5 -2.66454e-15 2
  name "wall(49)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -0.5 0 1
  name "wall(17)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -0.5 0 -1
  name "wall(33)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -1.5 0 -1
  name "wall(54)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -1.5 0 -2
  name "wall(26)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -0.5 0 -2
  name "wall(27)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 1.5 0 -3
  name "wall(30)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 0.5 0 -3
  name "wall(56)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -1.5 0 1
  name "wall(16)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 3.5 0 1
  name "wall(34)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 0.5 0 -4
  name "wall(36)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 3.5 0 -4
  name "wall(44)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 2 0 0.5
  rotation 0 1 0 -1.5707953071795862
  name "wall(35)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 2 0 -0.5
  rotation 0 1 0 -1.5707953071795862
  name "wall(55)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 2.5 -1.77636e-15 -4
  name "wall(39)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 1.5 -1.77636e-15 -4
  name "wall(45)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 3.5 0 2
  name "wall(40)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 2.5 8.88178e-16 2
  name "wall(57)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 3.5 0 4
  name "wall(23)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -0.5 3.55271e-15 -4
  name "wall(37)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 2.5 0 1
  name "wall(32)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 1.5 0 4
  name "wall(46)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 2.5 1.77636e-15 4
  name "wall(47)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation 0.5 0 4
  name "wall(7)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -2 5.32907e-15 3.5
  rotation 0 1 0 -1.5707953071795862
  name "wall(48)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -4 0 2.5
  rotation 0 1 0 1.5708
  name "wall(2)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -4 5.15143e-14 1.5
  rotation 0 1 0 1.5708
  name "wall(9)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -4 -5.86198e-14 0.5
  rotation 0 1 0 1.5708
  name "wall(10)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -4 -5.15143e-14 -2.5
  rotation 0 1 0 1.5708
  name "wall(19)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -4 -7.10543e-15 -3.5
  rotation 0 1 0 1.5708
  name "wall(20)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -4 0 -0.5
  rotation 0 1 0 1.5708
  name "wall(12)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -4 0 -1.5
  rotation 0 1 0 1.5708
  name "wall(14)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
Wall {
  translation -4 1.06581e-14 3.5
  rotation 0 1 0 1.5708
  name "wall(22)"
  size 1 0.4 0.02
  appearance Roughcast {
    colorOverride 1 1 0
    textureTransform TextureTransform {
      scale 1 2.4
    }
  }
}
